src/platforms/java/common/enriching-events/attachments.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: "Attachments Preview"
```

Comment:
Android and Java and even Apple could reuse the includes

Reply:
I could make use of `<PlatformContent includePath="add-attachment" />`, but then native and Javascript also need to change it, but they have slightly different API, so we can't unify it yet.

Reply:
add-attachment is only the snippets, what's about creating its own mobile snippets and add support to the comments attachments?

https://github.com/getsentry/sentry-docs/blob/master/src/platforms/common/enriching-events/attachments/index.mdx#uploading-attachments

Reply:
Sorry @marandaneto I don't fully understand what you mean.

Reply:
the common content already exists, so I don't see why we can't use it.

https://docs.sentry.io/platforms/javascript/enriching-events/attachments/

Reply:
I know added includes. Better now @marandaneto ?